settings.gradle

```diff
@@ -3,7 +3,7 @@ enableFeaturePreview('GRADLE_METADATA')
 include 'atomicfu'
 include 'atomicfu-common'
 include 'atomicfu-js'
-include 'atomicfu-native'
+//include 'atomicfu-native'
```

Comment:
oops

Reply:
TBD. Native needs to be supported before we can merge it to master.